driver_config.yml

```diff
@@ -1,4 +1,5 @@
 cuda:
+  # renovate: datasource=custom.nvidia-driver depName=nvidia-cuda-driver
```

Comment:
# renovate: datasource=custom.nvidia-driver depName=nvidia-cuda-driver versioning:loose

Reply:
versioning=loose right?

Reply:
looks like they are using semver no ?

Reply:
Renovate did not parse it properly when I had it set to semver. Checked https://docs.renovatebot.com/modules/versioning/loose/ and https://docs.renovatebot.com/modules/versioning/semver/. Nvidia drivers don't follow strict semver 2.0 guidelines like rule 8, for instance: https://semver.org/.

Haven't checked renovate semver validation code though, but the docs do mention that following semver 2.0 is required.

Reply:
the default is: https://docs.renovatebot.com/modules/versioning/semver-coerced/. which is supposed to be forgiving